tests/Unit/PointwiseFunctions/Hydro/InitialData/IrrotationalBns.py

```diff
@@ -0,0 +1,73 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+import numpy as np
+
+
+def rotational_shift(shift, spatial_rotational_killing_vector):
+    return shift + spatial_rotational_killing_vector
+
+
+def rotational_shift_stress(rotational_shift, lapse):
+    return np.outer(rotational_shift, rotational_shift) / lapse**2
+
+
+def derivative_rotational_shift_over_lapse(
+    rotational_shift,
+    deriv_of_shift,
+    lapse,
+    deriv_of_lapse,
+    deriv_of_spatial_rotational_killing_vector,
+):
+    return (
+        deriv_of_shift + deriv_of_spatial_rotational_killing_vector
+    ) / lapse - np.outer(deriv_of_lapse / lapse**2, rotational_shift)
+
+
+def divergence_rotational_shift_stress(
```

Comment:
The c++ part of the test for this function is missing.

Reply:
Oh that function was removed because it became unnecessary, I'll remove it from the python functions, and from the `IrrotationalBns.cpp` file